<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="19771" started-at="2016-08-26T11:20:54Z" finished-at="2016-08-26T11:21:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="19771" started-at="2016-08-26T11:20:54Z" finished-at="2016-08-26T11:21:14Z">
      <class name="seleniumGomez.TestClass">
        <test-method status="PASS" signature="Setup()[pri:0, instance:seleniumGomez.TestClass@b97c004]" name="Setup" is-config="true" duration-ms="10088" started-at="2016-08-26T19:20:54Z" finished-at="2016-08-26T19:21:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="pageTest()[pri:0, instance:seleniumGomez.TestClass@b97c004]" name="pageTest" duration-ms="6544" started-at="2016-08-26T19:21:04Z" finished-at="2016-08-26T19:21:11Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<3> but was:<2>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<3> but was:<2>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at seleniumGomez.TestClass.pageTest(TestClass.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:seleniumGomez.TestClass@b97c004]" name="TearDown" is-config="true" duration-ms="3071" started-at="2016-08-26T19:21:11Z" finished-at="2016-08-26T19:21:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- seleniumGomez.TestClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
